haproxy/README.md

```diff
@@ -49,8 +50,27 @@ create your own configmap and values.
 | `ingress.tls.secretName`| Name of the secret containing the TLS certificate and key |
 | `haproxy.frontendPort` | Port for the Proxy to listen on the frontend | 30636
 | `haproxy.config`       | Suffix of the used config-map | ldap-tls
-| `haproxy.ldapTls.backend` | Service the proxy should connect to | none
-| `haproxy.ldapTls.certificatePath` | Path to the corresponding root-certificate | /etc/ssl/certs/ca-certificates.crt   
 | `nodeSelector` | Pod node selector | `{}`
 | `tolerations` | Pod tolerations | `[]`
 | `affinity` | Pod affinity rules | `{}`
+
+### ldap-tls
+| Parameter              | Description            | Default
+|---                     | ---                    | ---
+| `haproxy.config`       | Suffix of the used config-map | ldap-tls
+| `haproxy.ldapTls.backend` | Service the proxy should connect to | none
+| `haproxy.ldapTls.certificatePath` | Path to the corresponding root-certificate | /etc/ssl/certs/ca-certificates.crt
+
+### galera
+
+| Parameter              | Description            | Default
+|---                     | ---                    | ---
+| `haproxy.galera.balance` | What balance mode HAProxy should use | source
```

Comment:
Please add the `haproxy.config` parameter here.

Reply:
I removed it from the `ldap-tls` section as it is a general parameter and added documentation on how the parameter should be used.